CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing (Code) to Python Community News
+
+Thank you for taking interest in contributing to this project.
+
+
+## Before you start
+This project has a [Code of Conduct](CODE_OF_CONDUCT.md) that you MUST follow in order to participate in this project in any way.
+
+This document is intended to help first-time or returning contributors to the project. It covers making contributions outside of the scheduled content and topics. For more information on how to submit a topic or conference, please see the [README](README.md).
+
+## About the code in this project
+
+You can break the code into essentially four parts:
+- [Contributing (Code) to Python Community News](#contributing-code-to-python-community-news)
+  - [Before you start](#before-you-start)
+  - [About the code in this project](#about-the-code-in-this-project)
+    - [For All Types of Contributions](#for-all-types-of-contributions)
+    - [Automation with Github via GH Actions](#automation-with-github-via-gh-actions)
+  - [General Administration around GitHub](#general-administration-around-github)
+    - [Deployting to our third-party platforms via API](#deployting-to-our-third-party-platforms-via-api)
+  - [Deploying of content to the web via Render Engine](#deploying-of-content-to-the-web-via-render-engine)
+
+You are able to contribute to all four of these parts but in different ways.
+
+### For All Types of Contributions
+To ensure that everyone is one the same page, please follow this process for all types of contributions.
+
+1. Read and follow [Code of Conduct](CODE_OF_CONDUCT.md)
+2. Create an issue on GitHub with the title of your perspective change. Use the approriate labels for the type of contribution you are making. Examples of appropriate labels are:
+    - `bug`
+    - `feature`
+    - `documentation`
+    - `gh action`
+
+    For bugs and features include a label for the area of the project that is affected. Examples of appropriate labels are:
+        - `newsletter`
+        - `website`
+        - `podcast`
+
+3. Specify that you will work on this issue. This lets others know that someone is actively working on the project.
+4. Fork the repository on your profile and make changes locally. 
+
+#### If Wanting to Work on an Existing Issue
+
+**Claim** the issue by commenting on it or stating you will work on the issue. We will accept PRs from the first person (collaborating on an issue is encouraged).
+
+![Create a Branch from an Issue](.github/assets/Create%20Branch.jpg)
+
+### Automation with Github via GH Actions
+
+GH Actions allow you to automate the deployment of your code to GitHub.
+
+We intend to use GitHub Actions to automate the compilation of issues to consider for the show and the creation of at show pull request.
+
+We'll also use GitHub Actions to interact with issues as they relate to the associated [Project](https://github.com/users/kjaymiller/projects/4) boards.
+
+Just adding a missing period here.
```

Comment:
```suggestion
```

Sorry, that was meant to be a comment and not part of the suggestion. I must have accidentally put it inside of the suggestion block instead of after.